packages/models/src/article/Article/behaviors/api.ts

```diff
@@ -0,0 +1,32 @@
+import { ArticleBehaviors } from "./types.js";
+import {
+  MittwaldAPIV2Client,
+  assertStatus,
+  extractTotalCountHeader,
+} from "@mittwald/api-client";
+
+export const apiArticleBehaviors = (
+  client: MittwaldAPIV2Client,
+): ArticleBehaviors => ({
+  find: async (id) => {
+    const response = await client.article.getArticle({
+      articleId: id,
+    });
+
+    if (response.status === 200) {
+      return response.data;
+    }
+    assertStatus(response, 429);
```

Comment:
```suggestion
    assertStatus(response, 429);
```

429 – Too many requests ???